docs/source/events/examples/payment-status-change.yaml

```diff
@@ -17,5 +17,17 @@ currency_code: EUR
 # Payment method specific data (stripe intent ID in this case)
 payment_data:
   transaction_id: pi_3NOKIODvbS6ezf4w21zIR6Ft
+# Line items:
+line_items:
```

Comment:
Should currency be here?

Reply:
No, IST 1.0 only allows one currency for all the line items.